editor/editor_about.cpp

```diff
@@ -154,7 +154,7 @@ EditorAbout::EditorAbout() {
 	Label *about_text = memnew(Label);
 	about_text->set_v_size_flags(Control::SIZE_SHRINK_CENTER);
 	about_text->set_text(String::utf8("\xc2\xa9 2014-present ") + TTR("Godot Engine contributors") +
```

Comment:
Could you also add a period at the end of this one? Seems I forgot it.

Reply:
`TTR("Godot Engine contributors") + "."` or `TTR("Godot Engine contributors.")`? :)

Reply:
I guess the former as some languages also use a different character for period.